tests/suites/test_suite_pk.function

```diff
@@ -299,6 +299,38 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void pk_invalid_param()
+{
+    mbedtls_pk_context ctx;
+    mbedtls_md_type_t md_alg_none = MBEDTLS_MD_NONE;
+    mbedtls_pk_type_t pk_type = 0;
+    unsigned char buf[] = {0x01, 0x02, 0x03, 0x04, 0x05, 0x06};
```

Comment:
This should have a space after the opening brace and before the closing brace to match code style.
```suggestion
    unsigned char buf[] = { 0x01, 0x02, 0x03, 0x04, 0x05, 0x06 };
```

Reply:
Sorry missed that. Added spaces now